conf/deployment/fusion_aas/managed_3az_provider_qe_3m_3w_m52x.yaml

```diff
@@ -0,0 +1,31 @@
+---
+ENV_DATA:
+  platform: 'fusion_aas'
+  deployment_type: 'managed'
+  # MS-provider cluster via rosa _addons
+  # or via rosa_service i.e appliance mode
+  appliance_mode: True
```

Comment:
Isn't the appliance mode no longer is use with FusionaaS?

Reply:
Right, it will be removed